sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/CollationBenchmark.scala

```diff
@@ -185,6 +185,48 @@ abstract class CollationBenchmarkBase extends BenchmarkBase {
     }
     benchmark.run(relativeTime = true)
   }
+
+  def benchmarkInitCap(
+      collationTypes: Seq[String],
+      utf8Strings: Seq[UTF8String]): Unit = {
+    type collationType = Int
```

Comment:
It is a collation id, and types should begin from an upper case letter.

Reply:
Thank you. Fixed